packages/web3-validator/src/keywords/eth.ts

```diff
@@ -0,0 +1,92 @@
+import { AnySchemaObject, FuncKeywordDefinition, SchemaCxt } from 'ajv';
+import { Web3ValidatorError } from '../errors';
+import { DataValidateFunction, DataValidationCxt, ValidInputTypes } from '../types';
+import {
+	isBoolean,
+	isString,
+	isBytes,
+	isInt,
+	isUInt,
+	isAddress,
+	isValidEthType,
+} from '../validation';
+
+export const metaSchema = {
+	title: 'Web3 Ethereum Compatible Types',
+	type: 'string',
+};
+
+const compile = (
+	type: string,
+	parentSchema: AnySchemaObject,
+	it: SchemaCxt,
+): DataValidateFunction => {
+	const typePropertyPresent = Object.keys(parentSchema).includes('type');
+
+	if (typePropertyPresent) {
+		throw new Web3ValidatorError([
+			{
+				keyword: 'eth',
```

Comment:
Could we expand `eth` to `ethType` for readability?

Reply:
I prefer it using `eth` to be concise as its gonna be repeated some places. Let's see what other think about it.